src/software/ai/navigator/path_planner/path_planner.h

```diff
@@ -15,8 +17,21 @@
 class PathPlanner
 {
    public:
-    virtual std::optional<std::vector<Point>> findPath(const Point &start,
-                                                       const Point &dest) = 0;
+    /**
+     * Returns a "good" path between start and dest.
```

Comment:
can we define what "good" means

Reply:
haha I knew someone would take issue with this. "good" depends on the what the path planner is trying to do, e.g. straightline vs spline has different ideas of what a good path is. Any suggestions for improving this comment?

Reply:
Is there an invariant  between all paths from any path planner you could define as good?

Reply:
Yeah, all the interface says is that this will return a path. I'd change it to just `Returns a path between start and dest`, because that's all the interface really requires. (_consider the case where we have a "StraightLinePathPlanner" for testing purposes_)

Reply:
Done

Reply:
I think this comment is resolved from my end, but it's @akhilveeraghanta's comment, so I'll let him close it.